site/content/blogs/release-v130.ja.md

```diff
@@ -0,0 +1,131 @@
+---
+title: 'AWS Copilot v1.30: `copilot run local` コマンド、Ctrl-C 機能、デプロイ前後処理、`copilot deploy` の機能拡張'
+twitter_title: 'AWS Copilot v1.30'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.30: `copilot run local` コマンド、Ctrl-C 機能、デプロイ前後処理、`copilot deploy` の機能拡張
+
+投稿日: 2023 年 8 月 30 日
+
+AWS Copilot コアチームは Copilot v1.30 のリリースを発表します。
+
+本リリースにご協力いただいた [@Varun359](https://github.com/Varun359) に感謝します。
+私たちのパブリックな[コミュニティチャット](https://app.gitter.im/#/room/#aws_copilot-cli:gitter.im) は成長しており、オンラインでは 500 人以上、[GitHub](http://github.com/aws/copilot-cli/) では 3k 以上のスターを獲得しています 🚀。
+AWS Copilot へご支援、ご支持いただいている皆様お一人お一人に感謝をいたします。
+
+Copilot v1.30 ではより柔軟で効率的な開発を支援する大きな機能強化が行われました:
+
+- **`copilot run local`**: Copilot に新しい操作コマンドが追加され、ローカルで Service を実行できるようになりました! [詳細はこちらをご覧ください](#copilot-run-local)。
+- **Ctrl-C によるデプロイのロールバック**: もう完了まで待つ必要はありません！いつでもターミナルから CloudFormation デプロイをロールバックできるようになりました。[詳細はこちらをご覧ください](#roll-back-deployments-with-ctrl-c)。
+- **デプロイ前後の Pipline アクション**: Workload/Environment のデプロイ前後に、DB マイグレーション、統合テスト、その他のアクションを挿入できるようになりました。[詳細はこちらをご覧ください](#deployment-actions)。
+- **`copilot deploy` の機能拡張**: このコマンドのスコープや柔軟性について拡充しました。[詳細はこちらをご覧ください](#copilot-deploy-enhancements)。
+- **`--detach flag`**: CloudFormation スタックイベントの進行状況をターミナル上でスキップできるようにしました。[詳細はこちらをご覧ください](#use---detach-to-deploy-without-waiting)。
+
+??? note "AWS Copilot とは？"
+
+    AWS Copilot CLI は AWS 上でプロダクションレディなコンテナ化されたアプリケーションのビルド、リリース、そして運用のためのツールです。
+    開発のスタートからステージング環境へのプッシュ、本番環境へのリリースまで、Copilot はアプリケーション開発ライフサイクル全体の管理を容易にします。
+    Copilot の基礎となるのは、 AWS CloudFormation です。CloudFormation により、インフラストラクチャを 1 回の操作でコードとしてプロビジョニングできます。
+    Copilot は、さまざまなタイプのマイクロサービスの作成と運用の為に、事前定義された CloudFormation テンプレートと、ユーザーフレンドリーなワークフローを提供します。
+    デプロイメントスクリプトを記述する代わりに、アプリケーションの開発に集中できます。
+
+    より詳細な AWS Copilot の紹介については、[Overview](../docs/concepts/overview.ja.md) を確認してください。
+
+<a id="copilot-run-local"></a>
+## `copilot run local`
+Service に対する変更作業をしているときに `copilot run local` を使用すると、デプロイのオーバーヘッドなしにコードの変更をテストできるため、イテレーションループを高速化できます。開始するには、まず `copilot svc deploy` を実行して Service のバージョンをデプロイする必要があります。
+
+Service のデプロイが完了したら、コードの修正を開始できます。変更をテストする準備をした後、`copilot run local` を実行すると、Copilot がプライマリコンテナとサイドカーの両方に対して以下が実行されます。
```

Comment:
It seems to be difficult to determine the subject in this sentence because there are two particles "が".
I prefer "Copilot はプライマリコンテナとサイドカーコンテナの両方に対して以下を実行します" or something.